### PR TITLE
New Drinks n' Cocktails

### PR DESCRIPTION
**Pull request by nauticall**
Sizeable assortment of new drinks and cocktails for your viewing pleasure, as well as some tweaks to existing ones.

If you're a wiki fellow please contact me if you want the sprites for these drinks. Alternatively copy paste them from IconDiffBot because that works too, surprisingly.

## New Drinks & Spirits
- **Applejack**: The brandy of the apple world. Distilled hard cider. Used to make the Red Dwarf Sangria and the Forbidden Apple.
- **Triple Sec**: Orange peel liqueur. Used to make these new cocktails: Mendellian, Cosmopolitan, Forbidden Apple, as well as being part of the new, more faithful Margarita recipe (tequila + lime juice + triple sec).
- **Fernet**: Herbal bitter liqueur. Used to make Fernet con Coca.
- **Cranberry Juice**: It's cranberry juice. Currently solely used to make the new Cosmopolitan, although there are definitely more cocktails to be made with this.
- **Rose Wine**: Fruity rose wine from Silverport. Standalone currently.
- **Assunzione Wine**: Another artisan wine, this time from Assunzione. Used to make the Red Dwarf Sangria.

## New Cocktails
_All recipes are equal parts (1:1:1)._
- **Mendellian**: Blue curacao, triple sec, lime juice. Citrusy spin on the Cosmopolitan.
- **Cosmopolitan**: Triple sec, cranberry juice, lime juice. Previously unmakeable due to the lack of cranberry juice that is now here. 
- **Fernet con Coca**: (Comet) cola, fernet. _South American classic._
- **Forbidden Apple**: Applejack, champagne, triple sec. _Not strictly faithful to the original recipe but the original recipe doesn't even have apples, so why call it an apple?_
- **New Margarita Recipe**: Now equal parts tequila, lime juice, and (new) triple sec.
- **Red Dwarf Sangria**: Assunzione wine, orange juice, applejack. _Sweet and punchy._

Time to mix drinks and change lives.

### CODE REVIEW

**Comment by nauticall** [comment]
idb is dead :crab:

**Comment by SleepyGemmy** [comment]
!review

File: code/modules/reagents/reagent_containers/food/drinks/bottle.dm [hunk 1]
```diff
@@ -575,6 +589,24 @@
...
+	name = "La Cerise d'Eclat Triple Sec"
+	desc = "A fruity Xanu liqueur made from orange peels. Best mixed as part of cocktails."
+	desc_extended = DRINK_FLUFF_ZENGHU
+	icon_state = "triplesecbottle"
+	center_of_mass = list("x"=16, "y"=7)
+	reagents_to_add = list(/singleton/reagent/alcohol/triplesec = 100)
+
+
+
 // Soda
```
Comment (SleepyGemmy) [inline code/modules/reagents/reagent_containers/food/drinks/bottle.dm:610]:
```suggestion
	reagents_to_add = list(/singleton/reagent/alcohol/triplesec = 100)

// Soda
```

File: code/modules/reagents/reagent_containers/food/drinks/bottle.dm
Comment (SleepyGemmy) [inline code/modules/reagents/reagent_containers/food/drinks/bottle.dm:1]: named/brand bottles should have `\improper` in front of them, so examining says "This is a Cool Bottle." and not "This is Cool Bottle."

Comment (nauticall) [reply code/modules/reagents/reagent_containers/food/drinks/bottle.dm:1]: would make everything else inconsistent (like three bottles have `\improper`) and making every bottle say `\improper` would be out of scope IMO. But if you insist, I can go do it.

Comment (SleepyGemmy) [reply code/modules/reagents/reagent_containers/food/drinks/bottle.dm:1]: talked about on discord, considering resolved.

Comment (nauticall) [reply code/modules/reagents/reagent_containers/food/drinks/bottle.dm:1]: i yield...

**Review by SleepyGemmy — APPROVED** [review]

**Comment by chaoticidealism** [comment]
> idb is dead 🦀

yeah :( Anybody know if there's another similar option?

**Review by NonQueueingMatt — APPROVED** [review]

**Pull request merged**